source/client-side-encryption/tests/README.rst

```diff
@@ -2490,7 +2506,7 @@ The following tests that a mongocryptd client is not created when shared library
    .. code:: javascript
 
       {
-        "mongocryptdURI": "mongodb://localhost:27021"
+        "mongocryptdURI": "mongodb://localhost:27021/db?serverSelectionTimeoutMS=1000"
```

Comment:
This change speeds up the test in a situation when the test fails. Tests in https://github.com/mongodb/specifications/tree/master/source/client-side-encryption/tests#bypass-spawning-mongocryptd already do this.